### PR TITLE
Breath holding task + bug fixes

### PR DESCRIPTION
**Pull request by esavary**
- Fixes the lag in the movie
- Changes the movie filename
- Adds breath_holding_task

### CODE REVIEW

File: control_task.psyexp [hunk 1]
```diff
@@ -15,18 +15,18 @@
...
     <Param val="True" valType="bool" updates="None" name="Save psydat file"/>
-    <Param val="False" valType="bool" updates="None" name="Save wide csv file"/>
+    <Param val="True" valType="bool" updates="None" name="Save wide csv file"/>
     <Param val="1" valType="num" updates="None" name="Screen"/>
     <Param val="True" valType="bool" updates="None" name="Show info dlg"/>
     <Param val="False" valType="bool" updates="None" name="Show mouse"/>
     <Param val="norm" valType="str" updates="None" name="Units"/>
     <Param val="2022.3.0dev6" valType="str" updates="None" name="Use version"/>
-    <Param val="[1280, 800]" valType="code" updates="None" name="Window size (pixels)"/>
+    <Param val="[1920, 1080]" valType="code" updates="None" name="Window size (pixels)"/>
```
Comment (oesteban) [inline control_task.psyexp:29]: Is this change necessary?

File: resting_state.psyexp [hunk 2]
```diff
@@ -103,7 +103,7 @@
         <Param val="duration (s)" valType="str" updates="None" name="stopType"/>
         <Param val="1200" valType="code" updates="constant" name="stopVal"/>
         <Param val="True" valType="bool" updates="None" name="syncScreenRefresh"/>
-        <Param val="from exp settings" valType="str" updates="None" name="units"/>
+        <Param val="norm" valType="str" updates="None" name="units"/>
```
Comment (oesteban) [inline resting_state.psyexp:106]: this has no effect, let's roll back to exp settings (which is `norm` already)
```suggestion
        <Param val="from exp settings" valType="str" updates="None" name="units"/>
```

File: resting_state.psyexp [hunk 3]
```diff
@@ -144,7 +144,7 @@
         <Param val="(0, 0)" valType="list" updates="constant" name="pos"/>
         <Param val="True" valType="bool" updates="None" name="saveStartStop"/>
         <Param val="circle" valType="str" updates="None" name="shape"/>
-        <Param val="(0.5, 0.6)" valType="list" updates="constant" name="size"/>
+        <Param val="(0.2, 0.2)" valType="list" updates="constant" name="size"/>
```
Comment (oesteban) [inline resting_state.psyexp:147]:
```suggestion
        <Param val="(0.05, 0.06)" valType="list" updates="constant" name="size"/>
```

File: resting_state.psyexp [hunk 4]
```diff
@@ -167,7 +167,7 @@
         <Param val="text" valType="code" updates="None" name="name"/>
         <Param val="" valType="num" updates="constant" name="opacity"/>
         <Param val="0" valType="num" updates="constant" name="ori"/>
-        <Param val="(0, 0)" valType="list" updates="constant" name="pos"/>
+        <Param val="(0, -0.2)" valType="list" updates="constant" name="pos"/>
```
Comment (oesteban) [inline resting_state.psyexp:170]: ? why this?

File: trial_select.csv [hunk 5]
```diff
@@ -0,0 +1,34 @@
+trialNum,trialSelect,trialType,trialStart,trialEnd
```
Comment (oesteban) [inline trial_select.csv:1]: did I not add this file originally?

File: resting_state.psyexp (hunk 4 above)
Comment (esavary) [reply resting_state.psyexp:170]: Without this shift the circle and the text are superimposed

File: control_task.psyexp (hunk 1 above)
Comment (esavary) [reply control_task.psyexp]: No, I rechanged the windows size to [1280, 800].

**Review by oesteban — APPROVED** [review]
Please merge it yourself when you consider this is finished (I don't know whether you want to add some work you probably did today)
